docs/source/models/causes/neonatal/index.rst

```diff
@@ -135,8 +133,7 @@ unit time.
 .. graphviz::
 
     digraph NN_decisions {
-        graph [size="7,5"]
-        rankdir = LR;
+        rankdir = TB;
         lb [label="live birth", style=dashed]
         enn_alive [label="neonate survived\nfirst 7 days"]
         enn_dead [label="neonate died\nduring first 7 days"]
```

Comment:
If we wanted to be really explicit with the decision graph, we could add leaf nodes for "died of subcause k" for each subcause, and have the "simulant died" nodes point into these.

Reply:
Here is code that does something like this
```
.. graphviz::

    digraph NN_decisions {
        rankdir = TB;
        lb [label="live birth", style=dashed]
        enn_alive [label="neonate survived\nfirst 7 days"]
        enn_dead [label="neonate died\nduring first 7 days"]
        lnn_alive [label="neonate survived\nfirst 28 days"]
        lnn_dead [label="neonate died\nduring first 28 days"]
        died_of_subcause_1 [label="neonate died\nof subcause 1"]
        died_of_subcause_2 [label="neonate died\nof subcause 2"]
        died_of_subcause_K [label="neonate died\nof subcause K"]

        lb -> enn_alive  [label = "1 - enn_mr"]
        lb -> enn_dead [label = "enn_mr"]
        enn_alive -> lnn_alive [label = "1 - lnn_mr"]
        enn_alive -> lnn_dead [label = "lnn_mr"]
        enn_dead -> died_of_subcause_1
        lnn_dead -> died_of_subcause_1
        enn_dead -> died_of_subcause_2
        lnn_dead -> died_of_subcause_2
        enn_dead -> died_of_subcause_K
        lnn_dead -> died_of_subcause_K
    }
```

Unfortunately it looks terrible!
<img width="647" alt="image" src="https://github.com/user-attachments/assets/0e4f2962-4804-435b-bf91-c37da468c95f" />


Patches welcome...

Reply:
Since we're not insisting on trees, we could collapse the two "neonate died" nodes, [like this](https://edotor.net/?engine=dot#%0A%0Adigraph%20NN_decisions%20%7B%0A%20%20%20%20%20%20%20%20rankdir%20%3D%20TB%3B%0A%20%20%20%20%20%20%20%20lb%20%5Blabel%3D%22live%20birth%22%2C%20style%3Ddashed%5D%0A%20%20%20%20%20%20%20%20enn_alive%20%5Blabel%3D%22neonate%20survived%5Cnfirst%207%20days%22%5D%0A%20%20%20%20%20%20%20%20lnn_alive%20%5Blabel%3D%22neonate%20survived%5Cnfirst%2028%20days%22%5D%0A%20%20%20%20%20%20%20%20dead%20%5Blabel%3D%22neonate%20died%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_1%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%201%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_2%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%202%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_K%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%20K%22%5D%0A%0A%20%20%20%20%20%20%20%20lb%20-%3E%20enn_alive%20%20%5Blabel%20%3D%20%221%20-%20enn_mr%22%5D%0A%20%20%20%20%20%20%20%20lb%20-%3E%20dead%20%5Blabel%20%3D%20%22enn_mr%22%5D%0A%20%20%20%20%20%20%20%20enn_alive%20-%3E%20lnn_alive%20%5Blabel%20%3D%20%221%20-%20lnn_mr%22%5D%0A%20%20%20%20%20%20%20%20enn_alive%20-%3E%20dead%20%5Blabel%20%3D%20%22lnn_mr%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_1%20%5Blabel%3D%22p_1%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_2%20%5Blabel%3D%22p_2%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_K%20%5Blabel%3D%22p_K%22%5D%0A%20%20%20%20%7D%0A):

```dot
digraph NN_decisions {
        rankdir = TB;
        lb [label="live birth", style=dashed]
        enn_alive [label="neonate survived\nfirst 7 days"]
        lnn_alive [label="neonate survived\nfirst 28 days"]
        dead [label="neonate died"]
        died_of_subcause_1 [label="neonate died\nof subcause 1"]
        died_of_subcause_2 [label="neonate died\nof subcause 2"]
        died_of_subcause_K [label="neonate died\nof subcause K"]

        lb -> enn_alive  [label = "1 - enn_mr"]
        lb -> dead [label = "enn_mr"]
        enn_alive -> lnn_alive [label = "1 - lnn_mr"]
        enn_alive -> dead [label = "lnn_mr"]
        dead -> died_of_subcause_1 [label="p_1"]
        dead -> died_of_subcause_2 [label="p_2"]
        dead -> died_of_subcause_K [label="p_K"]
    }
```

<img width="575" alt="image" src="https://github.com/user-attachments/assets/c9ff137f-0fb8-4202-8d39-fcee6be84e92" />

Reply:
Of cours, I added three new variables that we'd have to define...

Reply:
Cool!  Do you know how to put ellipses between subcause 2 and subcause K?

Reply:
> Cool! Do you know how to put ellipses between subcause 2 and subcause K?

I have no idea

Reply:
Here's a [hack-y solution](https://edotor.net/?engine=dot#%0A%0Adigraph%20NN_decisions%20%7B%0A%20%20%20%20%20%20%20%20rankdir%20%3D%20TB%3B%0A%20%20%20%20%20%20%20%20lb%20%5Blabel%3D%22live%20birth%22%2C%20style%3Ddashed%5D%0A%20%20%20%20%20%20%20%20enn_alive%20%5Blabel%3D%22neonate%20survived%5Cnfirst%207%20days%22%5D%0A%20%20%20%20%20%20%20%20lnn_alive%20%5Blabel%3D%22neonate%20survived%5Cnfirst%2028%20days%22%5D%0A%20%20%20%20%20%20%20%20dead%20%5Blabel%3D%22neonate%20died%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_1%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%201%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_2%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%202%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_K%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%20K%22%5D%0A%0A%20%20%20%20%20%20%20%20%7Brank%3Dsame%3B%0A%20%20%20%20%20%20%20%20died_of_subcause_1%0A%20%20%20%20%20%20%20%20died_of_subcause_2%0A%20%20%20%20%20%20%20%20died_of_subcause_K%0A%20%20%20%20%20%20%20%20%7D%0A%0A%20%20%20%20%20%20%20%20lb%20-%3E%20enn_alive%20%20%5Blabel%20%3D%20%221%20-%20enn_mr%22%5D%0A%20%20%20%20%20%20%20%20lb%20-%3E%20dead%20%5Blabel%20%3D%20%22enn_mr%22%5D%0A%20%20%20%20%20%20%20%20enn_alive%20-%3E%20lnn_alive%20%5Blabel%20%3D%20%221%20-%20lnn_mr%22%5D%0A%20%20%20%20%20%20%20%20enn_alive%20-%3E%20dead%20%5Blabel%20%3D%20%22lnn_mr%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_1%20%5Blabel%3D%22p_1%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_2%20%5Blabel%3D%22p_2%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_K%20%5Blabel%3D%22p_K%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_2%20-%3E%20died_of_subcause_K%20%5Bstyle%3D%22dotted%22%2C%20arrowhead%3D%22none%22%5D%0A%20%20%20%20%7D%0A) based on [this answer](https://stackoverflow.com/questions/55508342/drawing-ellipsis-between-nodes-in-graphviz). There's probably a better way to draw an actual ellipsis than using a dotted edge, but I don't know what it is.

Oh, but here's an[ even more hack-y solution](https://edotor.net/?engine=dot#%0A%0Adigraph%20NN_decisions%20%7B%0A%20%20%20%20%20%20%20%20rankdir%20%3D%20TB%3B%0A%20%20%20%20%20%20%20%20lb%20%5Blabel%3D%22live%20birth%22%2C%20style%3Ddashed%5D%0A%20%20%20%20%20%20%20%20enn_alive%20%5Blabel%3D%22neonate%20survived%5Cnfirst%207%20days%22%5D%0A%20%20%20%20%20%20%20%20lnn_alive%20%5Blabel%3D%22neonate%20survived%5Cnfirst%2028%20days%22%5D%0A%20%20%20%20%20%20%20%20dead%20%5Blabel%3D%22neonate%20died%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_1%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%201%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_2%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%202%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_K%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%20K%22%5D%0A%0A%20%20%20%20%20%20%20%20%7Brank%3Dsame%3B%0A%20%20%20%20%20%20%20%20died_of_subcause_1%0A%20%20%20%20%20%20%20%20died_of_subcause_2%0A%20%20%20%20%20%20%20%20died_of_subcause_K%0A%20%20%20%20%20%20%20%20%7D%0A%0A%20%20%20%20%20%20%20%20lb%20-%3E%20enn_alive%20%20%5Blabel%20%3D%20%221%20-%20enn_mr%22%5D%0A%20%20%20%20%20%20%20%20lb%20-%3E%20dead%20%5Blabel%20%3D%20%22enn_mr%22%5D%0A%20%20%20%20%20%20%20%20enn_alive%20-%3E%20lnn_alive%20%5Blabel%20%3D%20%221%20-%20lnn_mr%22%5D%0A%20%20%20%20%20%20%20%20enn_alive%20-%3E%20dead%20%5Blabel%20%3D%20%22lnn_mr%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_1%20%5Blabel%3D%22p_1%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_2%20%5Blabel%3D%22p_2%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_K%20%5Blabel%3D%22p_K%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_2%20-%3E%20died_of_subcause_K%20%5Bcolor%3D%22white%22%2C%20label%3D%22...%22%5D%0A%20%20%20%20%7D%0A).

Reply:
Maybe [this](https://edotor.net/?engine=dot#%0A%0Adigraph%20NN_decisions%20%7B%0A%20%20%20%20%20%20%20%20rankdir%20%3D%20TB%3B%0A%20%20%20%20%20%20%20%20lb%20%5Blabel%3D%22live%20birth%22%2C%20style%3Ddashed%5D%0A%20%20%20%20%20%20%20%20enn_alive%20%5Blabel%3D%22neonate%20survived%5Cnfirst%207%20days%22%5D%0A%20%20%20%20%20%20%20%20lnn_alive%20%5Blabel%3D%22neonate%20survived%5Cnfirst%2028%20days%22%5D%0A%20%20%20%20%20%20%20%20dead%20%5Blabel%3D%22neonate%20died%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_1%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%201%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_2%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%202%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_K%20%5Blabel%3D%22neonate%20died%5Cnof%20subcause%20K%22%5D%0A%0A%20%20%20%20%20%20%20%20%7Brank%3Dsame%3B%0A%20%20%20%20%20%20%20%20died_of_subcause_1%0A%20%20%20%20%20%20%20%20died_of_subcause_2%0A%20%20%20%20%20%20%20%20died_of_subcause_K%0A%20%20%20%20%20%20%20%20%7D%0A%0A%20%20%20%20%20%20%20%20lb%20-%3E%20enn_alive%20%20%5Blabel%20%3D%20%221%20-%20enn_mr%22%5D%0A%20%20%20%20%20%20%20%20lb%20-%3E%20dead%20%5Blabel%20%3D%20%22enn_mr%22%5D%0A%20%20%20%20%20%20%20%20enn_alive%20-%3E%20lnn_alive%20%5Blabel%20%3D%20%221%20-%20lnn_mr%22%5D%0A%20%20%20%20%20%20%20%20enn_alive%20-%3E%20dead%20%5Blabel%20%3D%20%22lnn_mr%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_1%20%5Blabel%3D%22p_1%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_2%20%5Blabel%3D%22p_2%22%5D%0A%20%20%20%20%20%20%20%20dead%20-%3E%20died_of_subcause_K%20%5Blabel%3D%22p_K%22%5D%0A%20%20%20%20%20%20%20%20died_of_subcause_2%20-%3E%20died_of_subcause_K%20%5Bcolor%3D%22none%22%2C%20label%3D%22...%22%5D%0A%20%20%20%20%7D%0A) is better, using color="none" instead of color="white"